packages/web3-plugin-example/src/index.ts

```diff
@@ -0,0 +1,58 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+import { ContractAbi } from 'web3-eth-abi';
+import Contract from 'web3-eth-contract';
+import { Web3PluginBase } from 'web3-core';
+import { Address, Web3APISpec } from 'web3-types';
+// @ts-expect-error 'Web3' is declared but its value is never read.
+import { Web3 } from 'web3/dist/web3';
+
+import { AggregatorV3InterfaceABI } from './aggregator_v3_interface_abi';
+
+interface Price {
+	roundId: string;
+	answer: string;
+	startedAt: string;
+	updatedAt: string;
+	answeredInRound: string;
+}
+
+interface ChainlinkPluginAPI extends Web3APISpec {
+	getPrice: () => Promise<Price>;
+}
+
+declare module 'web3/dist/web3' {
```

Comment:
Have you tested if end user imports Web3 from 'web3' (instead of 'web3/dist/web3') will they see augmented API?

Reply:
After [this](https://github.com/ChainSafe/web3.js/pull/5393/commits/755d6d65fb490493b3fbf529589f27069e6e7f59) commit, `Web3` is now exported as a named export which means this is now:

```ts
import { Web3 } from 'web3';

declare module 'web3'
```